### PR TITLE
Illustrations for a rectangle and concentric circles with multiple circles

### PR DESCRIPTION
**Pull request by nangahamandine**
I have added files in the `example` folder that have:
- An illustration for a simple rectangle
- An illustration for concentric circles with multiple circles.

Find the results below:

Screenshot (215)
Screenshot (216)

### CODE REVIEW

**Comment by nangahamandine** [comment]
@Sudha247 please verify my illustrations based on issue #3 
I'd welcome whatever corrections you may have.

I'll proceed to work on the other illustrations that you listed while waiting for your reply

File: examples/concentric_circles.ml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    else
+      let c = Circle { c = center; radius } in
+      create_circles (n - 1) (radius + spacing) (c :: acc)
+  in
+  create_circles num_circles 10 []
+
+let show shapes = List.iter render_shape shapes
+
+let () =
+  Graphics.open_graph (" " ^ string_of_int (fst canvas_size) ^ "x" ^ string_of_int (snd canvas_size));
```
[the remark this answers is not in the document]
Comment (nangahamandine) [reply examples/concentric_circles.ml:30]: Thank you for the corrections. I'll make the changes right away!

[the remark this answers is not in the document]
Comment (AryanGodara) [reply examples/concentric_circles.ml:30]: It looks like you've completed most of the examples of shapes @nangahamandine 
Impressive progress 🎊

[the remark this answers is not in the document]
Comment (nangahamandine) [reply examples/concentric_circles.ml]: Thank you! I've completed all the examples and added other shapes. I'll open a new pull request now with the new changes

**Comment by nangahamandine** [comment]
Below are images of the newly added illustrations:
Screenshot (224)
Screenshot (225)
Screenshot (226)
Screenshot (227)
Screenshot (228)

**Comment by nangahamandine** [comment]
Apart from these illustrations, are there any other illustrations that you want me to do?

**Comment by AryanGodara** [comment]
> Apart from these illustrations, are there any other illustrations that you want me to do?

No need for that @nangahamandine . I think you've already made great progress here :)
It's nice to see that you're enthusiastic. Let's wait for @Sudha247 to take a proper look before merging this. But it's approved from my side.

Maybe you can start work on one of the new issues. Like tests

**Comment by nangahamandine** [comment]
Okay, awesome! Thank you for your guidance, I made all this progress thanks to the guidance of you and @Sudha247  :)

I'll start looking into other issues

**Review by Sudha247 — APPROVED** [review]
Looks great! Thanks.

**Comment by nangahamandine** [comment]
Thank you so much!!!

**Pull request merged**